### PR TITLE
interface resource

### PR DESCRIPTION
**Pull request by chris-rock**
- implement resource for linux und windows

### CODE REVIEW

File: lib/resources/interface.rb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  include Converter
+  def initialize(vulcano)
+    @vulcano = vulcano
+  end
+end
+
+class LinuxInterface < InterfaceInfo
+  def interface_info(iface)
+    # will return "[mtu]\n1500\n[type]\n1"
+    cmd = @vulcano.command("find /sys/class/net/#{iface}/ -type f -maxdepth 1 -exec sh -c 'echo \"[$(basename {})]\"; cat {} || echo -n' \\;")
```
Comment (chris-rock) [inline lib/resources/interface.rb:61]: Thanks to @arlimus for helping with this command :+1:

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by arlimus** [comment]
Another huge addition to the resource library, thank you @chris-rock !

**Comment by arlimus** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: ff4b5b58-4dbe-4acd-8a36-8f11e1a7db0d approved by: @arlimus

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**